src/SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
```

Comment:
ignore this - just realised that the provider solution doesn't use a shared assembly info file like the APIs do